conda/cli/conda_argparse.py

```diff
@@ -117,9 +117,10 @@ def do_call(arguments: argparse.Namespace, parser: ArgumentParser):
     """
     # First, check if this is a plugin subcommand; if this attribute is present then it is
     if getattr(arguments, "plugin_subcommand", None):
-        _run_command_hooks("pre", arguments.plugin_subcommand.name, sys.argv[2:])
+        plugin_arguments = sys.argv[2:]
```

Comment:
This makes the arguments immutable, which I think is what we need to have consistency between the various steps of the command flow
```suggestion
        plugin_arguments = tuple(sys.argv[2:])
```

Reply:
Below, for normal conda commands, we pass in an `argparse.Namespace` object. These are mutable. I will also update the code to pass in just a tuple there as well.

Plus, I'm now realizing how inconsistent that was too 🙈 (i.e. for built-in commands you get one thing, for plugin commands something totally different).

Thanks for catching this @kenodegard and @jezdez.